ajax/libs/openfl/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "openfl",
+  "npmName": "openfl",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "openfl.min.js",
+  "version": "7.0.0",
+  "description": "A fast, productive library for 2D cross-platform development.",
+  "keywords": [
+    "openfl",
+    "openfl.js",
+    "openfl-js",
+    "game",
+    "animation",
+    "sound",
+    "flash",
+    "graphics"
+  ],
+  "homepage": "http://www.openfl.org",
+  "license": "MIT",
+  "author": "Joshua Granick and other OpenFL contributors",
```

Comment:
Let's use
```js
"author": [
  "Joshua Granick",
  "other OpenFL contributors"
],
```

Assumed that you cloned cdnjs repo, in this case, please modify this and amend it into the commit 
If not, I will help you squash the commits 😊

Reply:
Thank you,

I've updated online, I did clone the repository initially, but I don't see it here, so I must have been using a different machine.

If you don't mind squashing, I appreciate the help :smile: